src/envo/envo-edit.owl

```diff
@@ -41722,7 +41722,7 @@ SubClassOf(<http://purl.obolibrary.org/obo/ENVO_03501192> <http://purl.obolibrar
 
 # Class: <http://purl.obolibrary.org/obo/ENVO_03501193> (pharmacy)
 
-AnnotationAssertion(Annotation(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> <pharmacy. (n.d.) American Heritage® Dictionary of the English Language, Fifth Edition. (2011). Retrieved August 11 2020 from https://www.thefreedictionary.com/pharmacy>) <http://purl.obolibrary.org/obo/IAO_0000115> <http://purl.obolibrary.org/obo/ENVO_03501193> "A healthcare facility which is used to prepare, dispense, or sell medical drugs."@en)
+AnnotationAssertion(Annotation(<http://purl.org/dc/elements/1.1/source> "American Heritage Dictionary of the English Language, Fifth Edition. (2011). Retrieved August 11 2020 from https://www.thefreedictionary.com/pharmacy"^^xsd:string) <http://purl.obolibrary.org/obo/IAO_0000115> <http://purl.obolibrary.org/obo/ENVO_03501193> "A healthcare facility which is used to prepare, dispense, or sell medical drugs."@en)
```

Comment:
@matentzn Is the "major bug" that the value of hasDbXref is an improperly formatted IRI? Or that hasDbXref is not supposed to have strings as values?

Reply:
> Is the "major bug" that the value of hasDbXref is an improperly formatted IRI?

Yes, it just caused a serious cascade of issues: https://github.com/protegeproject/protege/issues/1096

> that hasDbXref is not supposed to have strings as values?

Its not supposed to be an arbitrary string, it is supposed to be some kind of database identifier (DOI, PMID, ISBN etc); but that was not the problem here. The "major" problem was the IRI type.